aptos-move/framework/aptos-experimental/sources/trading/order_book/order_book_types.move

```diff
@@ -189,31 +189,23 @@ module aptos_experimental::order_book_types {
         OrderWithState::V1 { order, is_active }
     }
 
-    public fun tp_trigger_condition(take_profit: u64): TriggerCondition {
-        TriggerCondition::TakeProfit(take_profit)
+    public fun price_move_up_condition(price: u64): TriggerCondition {
+        TriggerCondition::PriceMoveAbove(price)
     }
 
-    public fun sl_trigger_condition(stop_loss: u64): TriggerCondition {
-        TriggerCondition::StopLoss(stop_loss)
+    public fun price_move_down_condition(price: u64): TriggerCondition {
+        TriggerCondition::PriceMoveBelow(price)
     }
 
     // Returns the price move down index and price move up index for a particular trigger condition
-    public fun index(self: &TriggerCondition, is_bid: bool):
+    public fun index(self: &TriggerCondition):
```

Comment:
Can we still make this change now?

Reply:
What do you mean?

Reply:
If the concern is breaking changes, its fine for now.

Reply:
Ah yes my major concern is breaking changes.